package.json

```
@@ -21,13 +21,13 @@
 		"test": "eslint ."
 	},
 	"peerDependencies": {
-		"mwbot": "^2.0.0",
-		"wdio-mediawiki": "^2.0.0",
-		"webdriverio": "^7.0"
+		"mwbot": "^2.1.0",
+		"wdio-mediawiki": "^2.2.0",
+		"webdriverio": "^8.16"
```

Updating this peer dependency to just `^8.16` indicates that the package no longer supports v7 which would require a major version bump of `wdio-wikibase`. We allow both by doing the following:

```suggestion
		"webdriverio": "^7.0 || ^8.16"
```

---

Makes absolute sense, thank you!